test/boost/secondary_index_test.cc

```diff
@@ -1416,7 +1415,7 @@ SEASTAR_TEST_CASE(test_secondary_index_on_non_pk_ck_column_and_aggregation) {
     // case of indexed_table_select_statement::do_execute.
 
     return do_with_cql_env_thread([] (cql_test_env& e) {
-        cql3::statements::set_internal_paging_size_guard g(page_scenarios_page_size);
+        e.db_config().select_internal_page_size.set(page_scenarios_page_size);
```

Comment:
We probably need to set the option on all shards, not only the local one, right?